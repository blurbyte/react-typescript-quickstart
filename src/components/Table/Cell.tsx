import styled from '@styles/styledComponents';
import { Colors } from '@styles/colors';

const Cell = styled.td`
  font-family: Helvetica, Arial, sans-serif;
  text-align: left;
  padding: 1.4rem;

  &:first-child {
    border-right: 1px solid ${Colors.gray};
  }

  & strong {
    font-weight: 700;
  }
`;

export default Cell;
