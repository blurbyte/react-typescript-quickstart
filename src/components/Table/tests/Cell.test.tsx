import * as React from 'react';
import { shallow } from 'enzyme';

import Cell from '../Cell';

describe('Cell', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Cell />);
    expect(wrapper).toMatchSnapshot();
  });
});
