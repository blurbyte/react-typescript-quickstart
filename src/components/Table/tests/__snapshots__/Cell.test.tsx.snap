// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell renders correctly 1`] = `
.c0 {
  font-family: Helvetica,Arial,sans-serif;
  text-align: left;
  padding: 1.4rem;
}

.c0:first-child {
  border-right: 1px solid #908e89;
}

.c0 strong {
  font-weight: 700;
}

<td
  className="c0"
/>
`;
